source/composer.d

```diff
@@ -90,6 +90,11 @@ class ComposeParser {
         }
         string stringContent = endChunk();
         match('"');
+        if (stringContent == r"\") {
```

Comment:
Danke für den Fix. Gebe dir recht, dass es für unsere Beispiele so funktioniert. Leider verhält sich der Parser damit leider intern nicht so wie erwartet. Bei diesen Escape-Zeichen ist die Parser-Position nämlich nach dem Aufruf von `quotedString` nicht mehr hinter dem letzten `"`. Wenn dahinter noch was käme, würde es also schief gehen. Das mag gerade nicht der Fall sein, aber langfristig baut man sich hier eine Stolperfalle.

Ich würde vorschlagen, den Commit aus dem PR erstmal wieder rauszunehmen (am besten mit Rebase, damit das Issue nicht geschlossen wird).

Reply:
Update: Das Thema ist mit c184ed33dd3a8475952bd002193d46548a4a99bb gelöst.

Reply:
Das braucht halt schon über 10% länger, wenn man für jedes gelesene Zeichen den String erweitert. Hatte zuvor denselben Fix ausprobiert und dachte mir, wozu, wenn der Compose-String nur aus einem Zeichen besteht (oder zumindest kein escaped quote enthält). Aber da bin ich wohl zu pragmatisch …

Zum PR: wenn ich den Commit rausnehme, verschwinden dann auch die darauf bezogenen Kommentare? Mach doch einen Cherry-Pick auf 52e3c45 und schließ den PR, das wär wohl ok so.

Reply:
Berechtigter Einwand. Dafür funktionieren jetzt auch die mehrzeiligen Klammern in `math.module`, das finde ich schon einen Mehrwert. Optimieren kann man das sicherlich noch, da bin ich gerne für Vorschläge offen.

Cherry-Pick mach ich so 👍